airbyte-integrations/connectors/destination-bigquery/src/main/kotlin/io/airbyte/integrations/destination/bigquery/BigQueryDestination.kt

```diff
@@ -83,23 +95,194 @@ class BigQueryDestination : BaseConnector(), Destination {
                     .setUseLegacySql(false)
                     .build()
 
+            val result = BigQueryUtils.executeQuery(bigquery, queryConfig)
+
+            if (result.getLeft() == null) {
+                throw ConfigErrorException(result.right)
+            }
+
             if (UploadingMethod.GCS == uploadingMethod) {
-                val status = checkGcsPermission(config)
-                if (status!!.status != AirbyteConnectionStatus.Status.SUCCEEDED) {
-                    return status
+
+                val GcsStatus = checkGcsAccessPermission(config)
```

Comment:
`GcsStatus` => `gcsStatus`. local variable should start with a lower case.

Reply:
sure, I have changed the variable name